product/roundhouse/consoles/DefaultConfiguration.cs

```diff
@@ -37,7 +37,7 @@ public sealed class DefaultConfiguration : ConfigurationPropertyHolder
         public string VersionTableName { get; set; }
         public string ScriptsRunTableName { get; set; }
         public string ScriptsRunErrorsTableName { get; set; }
-        public string EnvironmentName { get; set; }
+        public string EnvironmentNames { get; set; }

```

Comment:
Like DropkicK and other tools that use the API.  You need to provide a backwards compatible EnvironmentName as well.